esti/lakectl_test.go

```diff
@@ -856,3 +857,69 @@ func TestLakectlBranchProtection(t *testing.T) {
 	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" branch-protect add lakefs://"+repoName+" "+mainBranch, false, "lakectl_empty", vars)
 	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" branch-protect list lakefs://"+repoName, false, "lakectl_branch_protection_list.term", vars)
 }
+
+func TestLakectlAbuse(t *testing.T) {
+	repoName := generateUniqueRepositoryName()
+	storage := generateUniqueStorageNamespace(repoName)
+	vars := map[string]string{
+		"REPO":    repoName,
+		"STORAGE": storage,
+		"BRANCH":  mainBranch,
+	}
+	RunCmdAndVerifySuccessWithFile(t, Lakectl()+" repo create lakefs://"+repoName+" "+storage, false, "lakectl_repo_create", vars)
+
+	fromFile := ""
+	const totalObjects = 5
+	for i := 0; i < totalObjects; i++ {
+		vars["FILE_PATH"] = fmt.Sprintf("data/ro/ro_1k.%d", i)
+		fromFile = fromFile + vars["FILE_PATH"] + "\n"
```

Comment:
I don't understand this loop at all.
1. Each iteration appends an entire new line to `fromFile`!  How does that run?
1. What writes these 5 objects?  Do they need to be added to the repo?

Reply:
1. The commands read a file from the provided param and reads line by lines keys from the file
2. What writes these 5 objects is line 876 (now 875), we use an existing test file in our repo for the physical object

Reply:
OK, thanks!  The code is a bit confusing.  I would probably find it easier to read if you wrote the filename to the `abuse-read` file immediately, rather than build a string in-memory and then write it on l. 879.

Reply:
I agree but this requires us to handle stdin input in the lakectl tests... 😬